packages/core/src/request-builder/header-builder/headers-util.ts

```diff
@@ -0,0 +1,57 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { MapType } from '@sap-cloud-sdk/util';
+
+export function isNullish(x: any): x is null | undefined {
```

Comment:
It seems that there should be a build in method option to do that?

Reply:
There is the nullish coalscing operator but this is not helpful in many cases. Rambda has a function that does that, but I would like to get rid of rambda in the long run (at least for everything related to destination handling) if there is not good reason to keep it. I agree, that this doesn't belong here though and I will move it to util. We have similar functionality there.